yellowbrick/features/rankd.py

```diff
@@ -240,27 +275,173 @@ def rank(self, X, algorithm=None):
+        if is_dataframe(X):
+            X = X.as_matrix()
+
         return self.ranking_methods[algorithm](X)
 
-    def draw(self, X, **kwargs):
+    def finalize(self, **kwargs):
+        """
+        Finalize executes any subclass-specific axes finalization steps.
+        The user calls poof and poof calls finalize.
+
+        Parameters
+        ----------
+        kwargs: dict
+            generic keyword arguments
+
+        """
+        # Set the title
+        self.set_title(
+            "{} Ranking of {} Features".format(
+                self.ranking_.title(), len(self.features_)
+            )
+        )
+
+
+##########################################################################
+## Rank 1D Feature Visualizer
+##########################################################################
+
+class Rank1D(RankDBase):
+    """
+    Rank1D computes a score for each feature in the data set with
+    a specific metric or algorithm (e.g. takes the shapiro) then returns
+    them ranked as a bar plot.
+
+    Parameters
+    ----------
+    ax : matplotlib Axes, default: None
+        The axis to plot the figure on. If None is passed in the current axes
+        will be used (or generated if required).
+
+    algorithm : one of {'shapiro', }, default: 'shapiro'
+        The ranking algorithm to use, default is 'Shapiro-Wilk.
+
+    features : list
+        a list of feature names to use
+        If a DataFrame is passed to fit and features is None, feature
+        names are selected as the columns of the DataFrame.
+
+    colormap : string or cmap, default: 'RdBu_r'
+        optional string or matplotlib cmap to colorize lines
+        Use either color to colorize the lines on a per class basis or
+        colormap to color them on a continuous scale.
+
+    kwargs : dict
+        Keyword arguments that are passed to the base class and may influence
+        the visualization as defined in other Visualizers.
+
+    Examples
+    --------
+
+    >>> visualizer = Rank2D()
+    >>> visualizer.fit(X, y)
+    >>> visualizer.transform(X)
+    >>> visualizer.poof()
+    """
+    ranking_methods = {
+        'shapiro': lambda X: np.array([shapiro(x)[0] for x in X.T]),
+    }
+
+    def __init__(self, ax=None, algorithm='shapiro', features=None,
+                 colormap='RdBu_r', **kwargs):
+        """
+        Initialize the class with the options required to rank and
+        order features as well as visualize the result.
+        """
+        super(Rank1D, self).__init__(ax=None, algorithm=algorithm, features=features,
+                         colormap=colormap, **kwargs)
+
+    def draw(self, R, **kwargs):
+        """
+        Draws the heatmap of the ranking matrix of variables.
+        """
+        self.ax.bar(np.arange(len(R)), R, color=self.colormap[0])
+        self.ax.set_xticks(np.arange(len(R)))
+        self.ax.xaxis.grid(False)
+        self.ax.set_xticklabels(self.features_, rotation=90, ha='left')
```

Comment:
Tick labels seem to be a `finalize()` thing to me; and I have comments related to this in the `draw()` of Rank2D. I don't think any change is necessary right now, but I wanted to alert you to the fact that in the future some refactoring may occur along those lines. 

Also, thank you for adding the tick labels for both 1 and 2D, however there are a couple of scenarios that we should consider (and are the reason they were originally omitted from 2D). 

1. A user might not specify feature names 
2. There may be (and is usually the case) > 35 features 

In the first case, what will happen if you do `self.ax.set_xticklabels(None, rotation=90, ha='left')`? This is, after all the default case (and a good test to write!) 

In the second case, the features get jammed together and become unreadable. This was constantly happening to me with my models in Rank2D. We do need to figure something out, and I'm open to suggestions. For now, however (to get your PR merged quickly) I suggest the following:

1. Create an option in `RankDBase` for `show_feature_names` or something like that, which defaults to `True` but can be turned off if the labels get jammed up. 
2. Create an issue "Better feature name labeling for RankD methods" that discusses various strategies. 

Strategies include:

1. Showing on the first and last feature on all axes (in the Rank2D and if we ever have a Rank3D) 
2. Eliminating every other feature name from the axes after a certain number of dimensions 
3. Using interactive graphs with hover to show which feature/pair of features exist.